src/crystal/init_project/template/gitignore.ecr

```diff
@@ -0,0 +1,4 @@
+/.deps/
+/.deps.lock
```

Comment:
I think we want to keep the .lock in the repo, at least I understood it as having the same functionality as bundlers `Gemfile.lock`.